lib/elocaps/ui/pages/game_page/game_page.dart

```diff
@@ -26,7 +26,7 @@ class GamePage extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final isGameCreated = useState(false);
+    var isGameCreated = useState(false);
```

Comment:
You can declare it as final since only the state will be modified and not the variable